tests/tests_suite_test.go

```diff
@@ -118,6 +123,6 @@ var _ = ReportBeforeEach(func(specReport SpecReport) {
 	k8sReporter.JustBeforeEach(CurrentSpecReport())
 })
 
-var _ = ReportAfterEach(func(specReport SpecReport) {
+var _ = JustAfterEach(func() {
```

Comment:
note that ReportBeforeEach is now not before, so maybe include what it does here as well ?
(it just print one liner IIRC)